src/vs/base/browser/ui/positronComponents/splitters/verticalSplitter.tsx

```diff
@@ -220,130 +208,18 @@ export const VerticalSplitter = ({
 	 * Sash onPointerLeave handler.
 	 * @param e A PointerEvent that describes a user interaction with the pointer.
 	 */
-	const sashPointerLeaveHandler = (e: PointerEvent<HTMLDivElement>) => {
+	const sashPointerLeaveHandler = (e: React.PointerEvent<HTMLDivElement>) => {
 		// When not resizing, trigger the delayer.
-		if (!resizeParams) {
+		if (!resizing) {
 			hoveringDelayer.trigger(() => setHovering(false), hoverDelay);
 		}
 	};
 
-	/**
-	 * Sash onPointerDown handler.
-	 * @param e A PointerEvent that describes a user interaction with the pointer.
-	 */
-	const sashPointerDownHandler = (e: PointerEvent<HTMLDivElement>) => {
```

Comment:
This function has been replaced by `pointerDownHandler` - most of the logic is the same and just been moved over. I may end up renaming `pointerDownHandler` to `sashPointerDownHandler` if that's the preference.

Reply:
Let's rename `pointerDownHandler` to `sashPointerDownHandler` for consistency.